src/enums.js

```diff
@@ -67,13 +67,15 @@ export const cropLifeStage = enumify(['SEED', 'GROWING', 'GROWN'])
  * @enum {string}
  */
 export const itemType = enumify([
+  'CHOCOLATE',
   'COW_FEED',
   'CRAFTED_ITEM',
   'CROP',
   'FERTILIZER',
   'HUGGING_MACHINE',
   'MILK',
   'SCARECROW',
+  'SOY_MILK',
```

Comment:
Given the comments about removing the new `farmhand.item` entries, I think the changes to this file can be reverted.